modules/xfem/src/base/XFEM.C

```diff
@@ -59,7 +59,7 @@ XFEM::getCrackTipOrigin(std::map<unsigned int, const Elem* > & elem_id_crack_tip
   std::map<unsigned int, const Elem*> elem_id_map;
 
   int m = -1;
-  for (mit1 = _elem_crack_origin_direction_map.begin(); mit1 != _elem_crack_origin_direction_map.end(); mit1++)
+  for (; mit1 != _elem_crack_origin_direction_map.end(); ++mit1)
```

Comment:
I have other concerns after reviewing this particular piece of code! It is actually not used by anything that is checked in the repository right now, although we do need this for other features that are in the works.  We should address this with another issue.